packages/backend/src/build-disk-image.ts

```diff
@@ -344,7 +344,7 @@ export function createBuilderImageOptions(
       'bootc.build.image.location': imagePath,
       'bootc.build.type': type,
     },
-    Cmd: [image, '--type', type, '--target-arch', arch, '--output', '/output/'],
+    Cmd: [image, '--type', type, '--target-arch', arch, '--output', '/output/','--local'],
```

Comment:
```suggestion
    Cmd: [image, '--type', type, '--target-arch', arch, '--output', '/output/', '--local'],
```

I don't know if pr check is checking the format ? as I don't see any error

Reply:
ah, now I see them

Reply:
Fixed.